From dcae883b382f03a58a04d72f6a3bb446e4814ae9 Mon Sep 17 00:00:00 2001
From: Peter Rajnoha <prajnoha@redhat.com>
Date: Fri, 29 Aug 2025 13:50:20 +0200
Subject: [PATCH 46/47] tools: pvchange: unlock devices file at the end of cmd
 processing

Remove the line, not just comment out.

(cherry picked from commit d89869f17ec217ee758918047b7edba6c9aa017a)
---
 tools/pvchange.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/pvchange.c b/tools/pvchange.c
index d7376c499..1b43b0154 100644
--- a/tools/pvchange.c
+++ b/tools/pvchange.c
@@ -194,7 +194,6 @@ static int _pvchange_single(struct cmd_context *cmd, struct volume_group *vg,
 			log_error("Failed to set pvid for devices file.");
 		if (!device_ids_write(cmd))
 			log_warn("Failed to update devices file.");
-		//unlock_devices_file(cmd);
 	}
 
 	log_print_unless_silent("Physical volume \"%s\" changed", pv_name);
-- 
2.51.0

